### PR TITLE
Add WNP 126 (Fixes #14512)

### LINKED ISSUE #14512: WNP 126 EU + NA
Hello.

[HERE](https://docs.google.com/document/d/1AKws9Zwr6Y2miKqRu8JVzy56TTpL1ftIOnhS4smPm3Q/edit) is the brief doc for WNP 126. 

Go live is **May 14, 2024.**

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Adds WNP for Firefox 126 for NA and EU locales.

## Issue / Bugzilla link

#14512

## Testing

- http://localhost:8000/en-US/firefox/126.0/whatsnew/
- http://localhost:8000/en-GB/firefox/126.0/whatsnew/
- http://localhost:8000/de/firefox/126.0/whatsnew/
- http://localhost:8000/fr/firefox/126.0/whatsnew/
- http://localhost:8000/es-ES/firefox/126.0/whatsnew/
- http://localhost:8000/it/firefox/126.0/whatsnew/
- http://localhost:8000/pl/firefox/126.0/whatsnew/

### CODE REVIEW

**Comment by alexgibson** [comment]
This is code complete and ready for review, but do-not-merge still until we've had approvals signed off.

**Comment by alexgibson** [comment]
This has been approved, so ready to merge once reviewed.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx126-na.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_126_na') }}
+{% endblock %}
+
+{% block wnp_content %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-xl">
```
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx126-na.html:26]: **Todo:** Needs a closing div.

File: bedrock/firefox/tests/test_base.py [hunk 2]
```diff
@@ -837,6 +837,91 @@ def test_fx_125_0_0_pl(self, render_mock):
...
+        """Should use whatsnew-fx126-eu template for fr locale"""
+        req = self.rf.get("/firefox/whatsnew/")
+        req.locale = "fr"
+        self.view(req, version="126.0")
+        template = render_mock.call_args[0][1]
+        assert template == ["firefox/whatsnew/whatsnew-fx126-eu.html"]
+
+    @override_settings(DEV=True)
+    def test_fx_126_0_0_es_es(self, render_mock):
+        """Should use whatsnew-fx126-eu template for fr locale"""
```
Comment (stephaniehobson) [inline bedrock/firefox/tests/test_base.py:898]: **Nit:** Comment needs updating to en-ES

**Review by stephaniehobson — APPROVED** [review]
2 trivial changes.

R+  ✏️

**Comment by alexgibson** [comment]
Thanks @stephaniehobson, updated 👍

**Pull request merged**